synapse/models/infotech.py

```diff
@@ -338,6 +346,20 @@ def getModelDefs(self):
                     'doc': 'NIST NVD Common Weaknesses Enumeration Specification',
                     'ex': 'CWE-120',
                 }),
+
+                ('it:sec:tlp', ('int', {'enums': tlplevels}), {
+                    'doc': 'A Traffic-Light-Protocol enumeration. Levels defined by US CISA.',
```

Comment:
Do we want to expand the def to include additional CISA language? For example "... designation used to indicate information sharing permissions"?

Reply:
Trying to keep the definition short/concise but i'll see if i can fit in some of their language :+1: